flink-runtime-web/web-dashboard/src/app/pages/job/overview/flamegraph/job-overview-drawer-flamegraph.component.ts

```diff
@@ -50,12 +50,13 @@ import { JobLocalService } from '../../job-local.service';
   standalone: true
 })
 export class JobOverviewDrawerFlameGraphComponent implements OnInit, OnDestroy {
+  FlameGraphType = FlameGraphType;
   public isLoading = true;
   public now = Date.now();
   public selectedVertex: NodesItemCorrect | null;
   public flameGraph = {} as JobFlameGraph;
 
-  public graphType = 'on_cpu';
+  public graphType = FlameGraphType.ON_CPU;
```

Comment:
```suggestion
```

Reply:
The `graphType` is used in the html,  `nz-radio-button` will change the graph Type.